data/wanderers middle.txt

```diff
@@ -1823,15 +1823,15 @@ mission "Wanderers: Sestor Search: Human Spaceport"
 				`	"I don't want to jump to any conclusions."`
 					goto no
 					
-			`	The pilot seems to display an expression of distaste upon hearing of aliens. You try to describe the dynamic between the exiles, the Efreti, Mereti, and Sestor, but to no avail whatsoever. The pilot quickly loses interest in your explanation, so you decide to steer the conversation to an end.`
+			`	"Aliens!?" The pilot's face reddens, and his eyes narrow in anger. "Like those detestable squirrles, you mean? Told me they were peaceful, went further north, and nearly had my ship torn to pieces!" You try to tell him of the dynamic between the exiles, the Efreti, Mereti, and Sestor, but he clearly does not want to hear about the Kor, so you decide to steer the conversation to an end.`
```

Comment:
@Bladewood, I'll make a compromise between yours and @Pointedstick's version. He won't lose interest, but he won't rant either. Thanks for the good catch.